### ISSUE TITLE
[BUG]: Showing opcodes on clang with AArch64 target doesn't work 

### ISSUE THREAD

**Issue by xTachyon** [compiler-explorer/compiler-explorer#5442]
### Describe the bug

clang with the aarch64 target set and the "Compile to binary object" option will result in `??? <No assembly generated>` in the asm window.

ARM64 gcc 13.2 works fine with this option.

### Steps to reproduce

1. Select clang 16;
2. Add `-O2 -target aarch64` for command line;
3. Check `Compile to binary object`;
4. Success?

### Expected behavior

ARM64 opcodes are shown near the asm.

### Reproduction link

https://godbolt.org/z/MzejeaY6o

### Screenshots

![image](https://github.com/compiler-explorer/compiler-explorer/assets/16454588/92bb5dc7-678a-47a1-8239-86dd2a18bc3c)

**Comment by OfekShilon** [comment]
I do see this on the live site but am afraid I can't reproduce it on a local instance.

**Comment by DavidSpickett** [comment]
I think this is because of this line:
https://github.com/compiler-explorer/compiler-explorer/blob/455d92916a9c22a3e2ae2f094e00b60d520efd44/etc/config/c%2B%2B.amazon.properties#L388

That `objdump` likely only supports x86, as the GNU tools tend to be single target. Using `llvm-objdump` is a possible solution as this will support the same targets as the clang in the same install would.

**Comment by DavidSpickett** [comment]
arm64 gcc works because it uses the objdump it comes bundled with:
https://github.com/compiler-explorer/compiler-explorer/blob/455d92916a9c22a3e2ae2f094e00b60d520efd44/etc/config/c%2B%2B.amazon.properties#L1783

**Comment by narpfel** [comment]
> Using `llvm-objdump` is a possible solution as this will support the same targets as the clang in the same install would.

I agree, compilers that can target multiple architectures should IMHO use `llvm-objdump`.

See also #8980, which is the same issue but for Rust.
